src/components/search/Search.component.tsx

```diff
@@ -6,8 +6,16 @@ import search from '@/assets/icons/search.svg';
 
 import styles from './search.module.css';
 
-const Search = ({ theme }: { theme: string }) => {
-    const nameSelect: string[] = ['Price (High - Low)', 'Price (Low- High)', 'Newest', 'Oldest'];
+const Search = ({
+    theme,
+    getProductsByCategory,
+    getSortByName,
+}: {
+    theme: string;
+    getProductsByCategory: (name: string) => void;
+    getSortByName: (name: string) => void;
+}) => {
+    const nameSelect: string[] = ['Price (High - Low)', 'Price (Low - High)', 'Newest', 'Oldest'];
 
     const [selectedSort, setSelectedSort] = React.useState<string>(nameSelect[0]);
```

Comment:
better write like this `import { useEffect, useState } from 'react';` and you can delete `React.`

Reply:
fixed